spyder/app/mainwindow.py

```diff
@@ -88,77 +83,44 @@
 from qtawesome.iconic_font import FontError
 
 
-#==============================================================================
+# ============================================================================
 # Proper high DPI scaling is available in Qt >= 5.6.0. This attibute must
 # be set before creating the application.
-#==============================================================================
+# ============================================================================
 from spyder.config.manager import CONF

```

Comment:
Please move this import to a more appropriate place.

Reply:
Suggestions?

Reply:
Fixed